### ISSUE TITLE
Links broken

### ISSUE THREAD

**Issue by LPerezPS** [sdrdis/jquery.flowchart#103]
Hello.

I'm trying to reload data from server using "setData", operators, inputs and outputs work but the links look broken at first.

![image](https://user-images.githubusercontent.com/26585387/51856897-7a42ad00-22f6-11e9-8862-53e09bb7185e.png)

Then if i move some operator the links show correctly.

![image](https://user-images.githubusercontent.com/26585387/51856969-afe79600-22f6-11e9-80c4-a1d51b8dfbc5.png)

I'm using AngularJS, execute the method using ng-init, and retrive data from the server with http post.

Thanks in advance.

**Comment by ronaldoscotti** [comment]
Did you discovered how to solve this @LPerezPS? I'm facing the same problem, but I'm using jQuery instead of Angular.

**Comment by ronaldoscotti** [comment]
Just to help whoever is having this issue in the future:

I was able to solve this using the `redrawLinksLayer` built in method, after the flowchart initialization.

**Comment by LPerezPS** [comment]
Hi @ronaldoscotti 

Sorry for dont replay early but im a little busy, no i cant find a solution to this moment and i try the built method but still show the links broken, what version of jquery are using? and how you implement the method?; im using this way:

**"Init"**
var $flowchart = $("#id_design");
$flowchart.flowchart("redrawLinksLayer");

**Load**
$flowchart.flowchart("setData", data);
$flowchart.flowchart("redrawLinksLayer");

**Comment by ronaldoscotti** [comment]
I'm using with jQuery v1.12.4, in a Wordpress instalation. I've tried to use the `redrawLinksLayer` directly after the flowchart declaration, but with no success, just like you.

But, my flowchart is placed inside a Bootstrap modal, so I placed the `redrawLinksLayer` in the open modal event, like this:

`$('#modal-id').on('shown.bs.modal', function () {`
`    $flowchart.flowchart('redrawLinksLayer');`
`});`

This way, every time I open the modal containing the flowchart, the links are drawed again.

**Comment by LPerezPS** [comment]
Yeah, im gonna think is the version cuz im using 3.3.1 and the plugins was update 2 years age soo if the redrawLinksLayer work for you that is nice :D

**Comment by huaiwen** [comment]
Well, I'm facing the same problem and the 
```
$flowchart.flowchart("setData", data);
$flowchart.flowchart("redrawLinksLayer");
```
does't work for me.
 :(

**Comment by sandeep-kr-kgp** [comment]
Try 
```
setTimeout( function() {
$flowchart.flowchart( "redrawLinksLayer" );
});
```
